samples/generated/react-embedded-auth-with-sdk/src/components/Profile/ProfileSection.jsx

```diff
@@ -48,7 +48,7 @@ const ProfileSection = () => {
     if (!profile) {
       fetchProfile();
     }
-  }, [oktaAuth, editing, profile, setProfile]);
+  }, [oktaAuth, editing, profile, setProfile, setCorsError]);
```

Comment:
**Unrelated to this PR**

Maybe we should revisit this setError logic in sample, technically cors error is undetectable programatically to prevent hacker attack.